cpp/oneapi/dal/BUILD

```diff
@@ -56,10 +57,18 @@ dal_algos(
     ]
 )
 
+dal_ios(
```

Comment:
Can we generalize `dal_algos` to not duplicate code in `dal_ios`, for example `dal_modules_group`?

Reply:
Done.